internal/crossdock/server/oneway/echo.go

```diff
@@ -91,3 +86,25 @@ func (o *onewayHandler) callHome(ctx context.Context, callBackAddr string, body
 		panic(fmt.Sprintf("could not make call back to client: %s", err))
 	}
 }
+
+type thriftHandler struct {
+	*onewayHandler
```

Comment:
The functions are attached on the pointer to `onewayHandler`, also makes it easier to initialize (see the construction of this in `server.go`).